app/controllers/admin/payments_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Admin
+  class PaymentsController < BaseAdminController
+    before_action :ensure_service_operator
+    before_action :find_payroll_run
+    before_action :find_payment
+
+    def delete
```

Comment:
We've used the word "remove" in the UI. Should we name this action that too so it matches and makes it all link up?

Reply:
That's much neater, ta 👍